bruce-gordon/garden-share-ui
# Refactor

#### What does  this PR do?
- Employs react-cookie to store user id during login
- Uses react-cookie to remove user id after logout
- Cleans up Sass with mixins
- Creates matching style across app
- Fixes async issues with accepting/rejecting offers for listings
- Adds images to product cards

#### Where should the reviewer start?
- All scss files
- actions.js
- component files that require user id (Logout, Homepage, ProductPage, NewListingPage, MyListings, MyOffers)

#### How should this be manually tested?
- After pulling down the branch, navigate to each view. 
- On ProductPage and NewListingPage, complete form to ensure it works.
- Click Log Out and check devTools to see that userId cookie has been removed.
- Review scss files

#### Any background context you want to provide?
- These changes were made as an extension on my Turing Capstone project after the assignment was officially submitted.  I added these changes individually.
